webpack.config.js

```diff
@@ -21,6 +21,10 @@ module.exports = {
         libraryTarget: 'umd',
         library: 'OSG'
     },
+    node: {
+       fs: "empty"
+    },
+    target: 'node',
```

Comment:
We should probably have a target for node and web using something like:
https://github.com/TheLarkInn/compare-webpack-target-bundles
https://stackoverflow.com/questions/35903246/how-to-create-multiple-output-paths-in-webpack-config
Then we can use both web and node